go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/canonical/go-sp800.90a-drbg v0.0.0-20210314144037-6eeb1040d6c3 // indirect
 	github.com/canonical/go-tpm2 v0.0.0-20210827151749-f80ff5afff61
 	github.com/canonical/tcglog-parser v0.0.0-20210824131805-69fa1e9f0ad2 // indirect
+	github.com/client9/misspell v0.3.4 // indirect
```

Comment:
This one can now be removed with my latest changes, sorry that I overlooked this. I will fix it after my next meeting but everyone is welcome to just do that